src/Stepper/index.js

```diff
@@ -1,5 +1,5 @@
+export Step from './Step';
+export StepButton from './StepButton';
+export StepContent from './StepContent';
+export StepLabel from './StepLabel';
 export Stepper from './Stepper';
```

Comment:
@newoga Have look at the index. We are now using https://github.com/leebyron/ecmascript-more-export-from. Does that makes sens?